units/pod_initializing.go

```diff
@@ -0,0 +1,105 @@
+package units
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/evergreen-ci/cocoa"
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/pod"
+	"github.com/evergreen-ci/utility"
+	"github.com/mongodb/amboy"
+	"github.com/mongodb/amboy/dependency"
+	"github.com/mongodb/amboy/job"
+	"github.com/mongodb/amboy/registry"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+)
+
+const podInitJobName = "pod-initializing-job"
```

Comment:
I would call this something along the lines of "create-pod". The existing convention with hosts is that hosts are "created" when they're spawned in EC2.

Can you rename this `createPodJobName`?